src/components/LowerLayout/GeneralInfo/GeneralInfo.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import global from '../../../styles.module.css';
+import classes from './GeneralInfo.module.css';
+
+const GeneralInfo = (props) => {
+  return (
+    <div className={global.container}>
+      {props.copy.details.map(copy => {
+        return (
+          <div className={classes.vessel} key={copy.heading}>
+            <h3 className={classes.heading}>{ copy.heading }</h3>
+            <div className={classes.content}>
+            <p>{ copy.firstParagraph }</p>
+            
+            { copy.lists.map(list => {
```

Comment:
Be careful when variables names are hard to distinguish from one another. Like `lists` and `list`. Perhaps something like `listsCollection` and `list` is easier to read.